### ISSUE TITLE
Twitter feed on website?

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/website#9]
@nunogit @egonw Regarding the BridgeDb website, do we want a twitter feed there? (like we have on the BiGCaT website).... We should post a bit more frequently I think, but it would be great if everything is linked together on the website.

**Comment by DeniseSl22** [comment]
Would also be nice to have a news feed (or we announce news items on twitter?), such as new development plans, new linksets etc.

**Comment by DeniseSl22** [comment]
I now added a link to the twitter account in the footer. But I think it will look nice if we can do it in a side-bar on the right (I found a menu option to do that). Let me know what you think!

**Comment by Chris-Evelo** [comment]
Agree. Although that means that we also need to make sure we actually do tweet from that account regularly

**Comment by Chris-Evelo** [comment]
Ohh and having both might be a bit of overkill, why not use the twitter feed to show the news?

**Comment by DeniseSl22** [comment]
@nunogit : if you could point me to the twitter-plugin you used for the Bigcat.unimaas website , then I could also use that for the bridgedb website.
